solutions/calculate_mean.py

```diff
@@ -0,0 +1,48 @@
+"""
+This module provides a function to calculate the mean (average) of a list of numbers.
+
+Author: @Abel Teka
+"""
```

Comment:
Great work, Abel. Due to Even's example, let's put date.

Reply:
Thanks, Viktoriya, okay will do that.